### PR TITLE
Testing that Behavior fields get updated into the HPA

### PR DESCRIPTION
**Pull request by muaazsaleem**
E2E tests for the HPA behavior field.

Depends on https://github.com/zalando-incubator/stackset-controller/pull/240

### CODE REVIEW

File: cmd/e2e/configure_hpa_test.go [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+	"github.com/stretchr/testify/require"
+	zv1 "github.com/zalando-incubator/stackset-controller/pkg/apis/zalando.org/v1"
+)
+
+// TestConfigureHPA tests Behavior is reflected when stackset is created
+func TestConfigureHPA(t *testing.T) {
+	t.Parallel()
+	stacksetName := "configured-hpa"
+	var stabilizationWindow int32 = 300
```
Comment (mikkeloscar) [inline cmd/e2e/configure_hpa_test.go:17]: Maybe use a different value than the default here for the sake of the test :)

Comment (muaazsaleem) [reply cmd/e2e/configure_hpa_test.go]: good point

File: cmd/e2e/configure_hpa_test.go [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+	fullFirstName := fmt.Sprintf("%s-%s", stacksetName, stackVersion)
+	stack, err := waitForStack(t, stacksetName, stackVersion)
+	require.NoError(t, err, "failed to create stack without stabilization")
+	require.Nil(t, stack)
+
+	_, err = waitForHPA(t, fullFirstName)
+	require.NoError(t, err, "failed to create HPA")
+
+	hpa, err := waitForHPA(t, fullFirstName)
+	require.NoError(t, err)
```
Comment (mikkeloscar) [inline cmd/e2e/configure_hpa_test.go:64]: `waitForHPA` is done twice?

**Comment by muaazsaleem** [comment]
👍

**Pull request merged**